### ISSUE TITLE
Compatibility with Cocoapods 0.36.0.beta.2

### ISSUE THREAD

**Issue by robertjpayne** [QueryKit/QueryKit#20]
Seems QueryKit doesn't like the latest beta of Cocoapods:

Gemfile:
```ruby
source "https://rubygems.org"
gem "cocoapods", "0.36.0.beta.2"
```

Podfile:
```
platform :ios, '8.0'
pod 'QueryKit'
```

Commands:
```
bundle install
bundle exec pod install
xcodebuild -workspace <name>.xcworkspace -scheme <name>
```

Output:
```
The following build commands failed:
	Ld /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/armv7/QueryKit normal armv7
	Ld /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QueryKit normal arm64
	CreateUniversalBinary /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/Pods/QueryKit.framework/QueryKit normal armv7\ arm64
	GenerateDSYMFile /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/Pods/QueryKit.framework.dSYM /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/Pods/QueryKit.framework/QueryKit
	Ld /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/*****.build/Debug-iphoneos/*****.build/Objects-normal/armv7/***** normal armv7
	Ld /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/*****.build/Debug-iphoneos/*****.build/Objects-normal/arm64/***** normal arm64
	CreateUniversalBinary /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/*****.app/***** normal armv7\ arm64
	GenerateDSYMFile /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/*****.app.dSYM /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/*****.app/*****
	PhaseScriptExecution Embed\ Pods\ Frameworks /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/*****.build/Debug-iphoneos/*****.build/Script-9C71411E2A50EC258CF1F152.sh
	Ld /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/*****.build/Debug-iphoneos/*****Tests.build/Objects-normal/armv7/*****Tests normal armv7
	Ld /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/*****.build/Debug-iphoneos/*****Tests.build/Objects-normal/arm64/*****Tests normal arm64
	CreateUniversalBinary /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/*****Tests.xctest/*****Tests normal armv7\ arm64
	GenerateDSYMFile /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/*****Tests.xctest.dSYM /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Products/Debug-iphoneos/*****Tests.xctest/*****Tests
```

Looking at the Pods build log:

```
duplicate symbol _OBJC_IVAR_$_QKAttribute._name in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKAttribute-277787A4A92A1D6F.o
duplicate symbol _OBJC_CLASS_$_QKAttribute in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKAttribute-277787A4A92A1D6F.o
duplicate symbol _OBJC_METACLASS_$_QKAttribute in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKAttribute-277787A4A92A1D6F.o
duplicate symbol _OBJC_IVAR_$_QKQuerySet._managedObjectContext in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_IVAR_$_QKQuerySet._entityDescription in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_IVAR_$_QKQuerySet._predicate in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_IVAR_$_QKQuerySet._sortDescriptors in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_IVAR_$_QKQuerySet._range in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_IVAR_$_QKQuerySet._resultsCache in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _QKQuerySetErrorDomain in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_CLASS_$_QKQuerySet in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
duplicate symbol _OBJC_METACLASS_$_QKQuerySet in:
    /Users/************/Library/Developer/Xcode/DerivedData/*****-annvlazmioqncwcrjreldnxhrmty/Build/Intermediates/Pods.build/Debug-iphoneos/Pods-QueryKit.build/Objects-normal/arm64/QKQuerySet-2C6DA287F3437EB3.o
ld: 12 duplicate symbols for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Looks like maybe just a double up on symbols? Unsure if this is related to QK or Cocoapods.

**Comment by robertjpayne** [comment]
Actually mind me looks like if you use the subspec `QueryKit/Swift` it works just fine.

Maybe some installation notes in the readme would help?

**Comment by kylef** [comment]
I'm glad you managed to find a workaround, `pod 'QueryKit'` should work. It appears to be a bug in CocoaPods causing some symbols to be exposed twice. I'll have to have a deeper look when I get a chance.

**Comment by kylef** [comment]
Thanks for reporting @robertjpayne I figured out the actual issue in the podspec and fixed it and release 0.9.1 of QueryKit. There is also an underlying bug in CocoaPods where the linter doesn't catch this particular problem.
